docs/content/get-started/build-workers.md

```diff
@@ -2,30 +2,52 @@
 
 As you will be launching a cloud deployment, you need to build out the code executables which will be run by SpatialOS servers - these are called [workers]({{urlRoot}}/content/glossary#worker).
 
-In the Unity Editor, you first need to make sure Burst compilation is **disabled** from **Jobs** > **Enable Burst Compilation**. Then you can build your workers from the SpatialOS menu by clicking **Build for cloud** > **All workers**.
-
-<%(Callout type="info" message="Building workers for the first time may take while. Why not make yourself a cup of tea while you wait?")%>
+1. You first need to make sure Burst compilation is set to disabled; in your Unity Editor, navigate to **Jobs** > **Enable Burst Compilation** and select **disabled**.
+2. Now you can build your workers from the SpatialOS menu by selecting **Build for cloud** > **All workers**. 
+(Shown below.) 
+<br/>
+<br/>![SpatialOS menu in Unity]({{assetRoot}}assets/unity-spatialos-menu.png)
+<br/>_The SpatialOS menu in the Unity Editor_
+<br/>
+<br/>**NOTE:** Building workers for the first time may take a while (about 10 minutes). Why not make yourself a cup of tea or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos) while you wait?
+<br/>
+<br/>
+3. **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. 
 
-![SpatialOS menu in Unity]({{assetRoot}}assets/unity-spatialos-menu.png)
+>**TIP:** You may get a number of warnings displayed in the Unity Editor Console window. You can ignore the warnings at this stage; use the message icons on the right-hand side of the Console window to set it to display only info and error messages so you can see only the relevant messages.
 
-> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`
+**If you encounter build errors:** 
+<br/>You might not have selected the build supports your game needs during your Unity setup.
 
-<%(Callout type="alert" message="If you encounter build errors, you might not have selected the build supports your game needs during your Unity setup. <br/><br/>
-* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/> <br/>
+* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.
```

Comment:
```suggestion
* You need **Linux** build support. This is because server-workers in a cloud deployment always run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.
```